docs/source/apps/quickstart.md

```diff
@@ -1,27 +1,17 @@
 # Quickstart
 
-The MarketPlace enables software vendors to register their applications onto the platform in order to make them available (i.e., executable) to users who purchased them.
+The MarketPlace enables users to register their own applications onto the platform so that these are purchasable and accessible by other users.
+This section provides a brief overview on the registration process and usage of MarketPlace applications.
 
-This section should give them a brief overview on the resources and materials involved in the application development, registration and exploitation processes.
+1. **Setting up the MarketPlace interface layer.** The MarketPlace defines a standard application API that dictates the way your application can be communicated via the MarketPlace. A preliminary step is therefore to set up a corresponding interface layer for your application. This entails the identification of the [capabilities](../capabilities) your application should support. A good starting point is the [get-started app](https://github.com/materials-marketplace/get-started-app).<br><br>
```

Comment:
```suggestion
1. **Setting up the MarketPlace interface layer.** The MarketPlace defines a standard application API that dictates the way your application can be communicated with via the MarketPlace. A preliminary step is therefore to set up a corresponding interface layer for your application. This entails the identification of the [capabilities](../capabilities) your application should support. A good starting point is the [get-started app](https://github.com/materials-marketplace/get-started-app).


```

Reply:
- I want to create extra space between the list items so that it's not so dense to the eye.
- "communicate with via" sounds a bit awkward.

Reply:
I don't think you can simply write "communicate", but need [a preposition](https://ell.stackexchange.com/questions/175313/what-preposition-is-particularly-followed-by-communicate-in-terms-of-conveying). The issue is the indirect speech with "your app can be communicated with". I think there is a pause between the _with_ and the _via_ when you read it.

Reply:
Ok, I see what you mean. Done.